keyboards/launchpad/config.h

```diff
@@ -45,6 +45,14 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* Locking resynchronize hack */
 #define LOCKING_RESYNC_ENABLE
 
+/* Underlight Configuration */
+#define RGB_DI_PIN F4
```

Comment:
This should probably go into `keymaps/default_rgb/config.h` rather than the base configuration for the launchpad. Reason being that the rgb add on is an option.

Reply:
Done!